src/patternfly/components/Tabs/tabs.scss

```diff
@@ -112,167 +126,305 @@
   max-width: 100%;
   overflow-x: auto;
   -webkit-overflow-scrolling: touch;
-
-  // Right border on last tab
-  .pf-c-tabs__item:last-of-type .pf-c-tabs__button::before {
-    border-right-width: var(--pf-c-tabs__item--BorderWidth);
-  }
 }
 
-// Tab items
+// Tabs item
 .pf-c-tabs__item {
+  display: flex;
   flex: none;
+  align-items: stretch;
+
+  // Current
+  &.pf-m-current {
+    .pf-c-tabs__button,
+    .pf-c-tabs__button:hover::after {
+      --pf-c-tabs__button--Color: var(--pf-c-tabs__item--m-current--c-tabs__button--Color);
+      --pf-c-tabs__button--after--BorderColor: var(--pf-c-tabs__item--m-current--c-tabs__button--after--BorderColor);
+    }
+  }
 
-  // Tab item hover state
-  &:not(.pf-m-current):hover .pf-c-tabs__button::after {
+  // If focussed and hovered
+  &:not(.pf-m-current) .pf-c-tabs__button:focus::after {
```

Comment:
I would expect hover styles to be applied if I hover, focused or not. Also I would expect the hover/focus styles to match. Was it a design decision for them to not?

Reply:
Focus ring is obscured in this context, so `outline-offset` is modified to be visible, which causes the focus and hover styles to conflict. 
<img width="487" alt="Screen Shot 2020-03-13 at 10 10 45 AM" src="https://user-images.githubusercontent.com/5385435/76628359-f7d57680-6512-11ea-84b6-9a2f496dcfe9.png">
<img width="378" alt="Screen Shot 2020-03-13 at 10 10 39 AM" src="https://user-images.githubusercontent.com/5385435/76628362-f7d57680-6512-11ea-8492-b3be2b207c7e.png">

@mcarrano @mceledonia wdyt?

Reply:
@mattnolting so are you saying that we have to choose between whether to expose the hover or the focus rectangle? Is that because the only visual effect of hover is to change the border? Does this happen anywhere else in the library or is it unique here because of the way we chose to design the tab hover states?